src/pages/Dashboard/Modals/SupplyWithdraw/index.tsx

```diff
@@ -81,8 +81,10 @@ export const SupplyWithdrawUi: React.FC<ISupplyWithdrawUiProps & ISupplyWithdraw
       ]
     : [];
 
-  const calculateNewSupplyAmount = (initial: BigNumber, amount: BigNumber) => initial.plus(amount);
-  const calculateNewBorrowAmount = (initial: BigNumber, amount: BigNumber) => initial.minus(amount);
+  const calculateNewSupplyAmount =
```

Comment:
I think this is more clearly written as two separate functions. Writing it as two functions introduces unnecessary logic and arguments. Was there an issue with using the individual functions. 

`calculateNewBorrowAmount` probably should be named `calculateNewWithdrawBalance`

Reply:
In both cases it's the supply balance you're calculating, which goes up if you're supplying more tokens or goes up if you're withdrawing some

Reply:
Yes, you are right, I was confused there. I still think they are nicer as two functions opposed to one function with a boolean flag

Reply:
Can do. What would you name them though?

Reply:
`calculateAdditionalSupplyAmount` and `calculateWithdrawalSupplyAmount`

Reply:
Indeed yeah, sorry this got messy. I merged a PR I shouldn't have and ended up messing up the PR order.